config/src/index.ts

```diff
@@ -13,8 +13,12 @@ import { CommonConfig, Config } from "./types";
 // eslint-disable-next-line import/no-mutable-exports
 let config: Config;
 
-const COMMON: CommonConfig = {
+export const COMMON: CommonConfig = {
```

Comment:
So this doesnt need to be exported, you can access this from `config.common`, everything should be accessed through the `config` variable